Domain/DataModels/Task.cs

```diff
@@ -8,6 +8,6 @@ public class Task
         public string Subject { get; set; }
         public bool IsComplete { get; set; }
         public Guid? AssignedMemberId { get; set; }
-        public Member? AssignedMember { get; set; }
+        public Member AssignedMember { get; set; }
```

Comment:
Since we have the whole `AssignedMember` we can remove `AssignedMemberId` I guess.